### DISCUSSION TITLE
dev-practice/2024/09/23/technical-debt

### DISCUSSION THREAD

**Discussion by giscus** [warmblood-kr/warmblood-kr.github.io#2] [Announcements]
# dev-practice/2024/09/23/technical-debt

기술부채라는 표현을 자주 듣곤 합니다. 그런데 가만히 보면 원래의 뜻과는 다른 의미로 사용되는 경우가 많아보입니다. 원래 의미는 조금 더 심오합니다. 원래는 어떤 의미였을까요?

https://warmblood-kr.github.io/dev-practice/2024/09/23/technical-debt.html

**Comment by toracle** [comment]
giscus로 댓글을 달 수 있도록 추가해봤습니다.
글에 대한 의견을 들려주세요.
